cmd/kubeadm/app/cmd/phases/uploadconfig.go

```diff
@@ -71,6 +71,11 @@ func NewUploadConfigPhase() workflow.Phase {
 		Short:   "Uploads the kubeadm and kubelet configuration to a ConfigMap",
 		Long:    cmdutil.MacroCommandLongDescription,
 		Phases: []workflow.Phase{
+			{
+				Name:           "all",
+				Short:          "Uploads all configuration to a config map",
+				RunAllSiblings: true,
```

Comment:
Add `InheritFlags: getUploadConfigPhaseFlags(),`

Reply:
actually yes, this this seems correct.
thanks @yagonobre